docs/ext/web.rst

```diff
@@ -30,6 +30,24 @@ To install, run::
     pip install Mopidy-API-Explorer
 
 
+Mopidy-Auto
+===========
+
+https://github.com/gotling/mopidy-auto
+
+Mopidy extension to automate music playback based on time of day.
+
+**Notice!** This extension reacts to the events *tracklist_changed*, *track_playback_ended* and *track_playback_resumed* to accomplish this so other web extension will not work as expeced when this extension is installed.
```

Comment:
Sorry for the delay, if you fixup these typos I'll merge. 
s/extension/extensions/
s/expeced/expected/